src/transforms/util/githubApiRequests.ts

```diff
@@ -0,0 +1,31 @@
+import { GitHubAPI } from "probot";
+import { LoggerWithTarget } from "probot/lib/wrap-logger";
+
+interface CompareCommitsPayload {
+	owner: string;
+	repo: string;
+	base: string;
+	head: string;
+}
+
+// Used to compare commits for builds and deployments so we can
+// obtain all issue keys referenced in commit messages.
+export const compareCommitsBetweenBaseAndHeadBranches = async (
+	payload: CompareCommitsPayload,
+	github: GitHubAPI,
+	logger?: LoggerWithTarget
```

Comment:
Would it make sense to make it mandatory? Seems it is passed in all calls... (For tests we could always pass `getLogger('test')`